tests/rbd/rbd_utils.py

```diff
@@ -699,6 +699,24 @@ def clean_up(self, **kw):
                     "--yes-i-really-really-mean-it".format(pool=pool)
                 )
 
+            # Deletes all test pools in the cluster
+            if flag == "all":
+                out = self.exec_cmd(
+                    cmd="ceph df detail --format json-pretty", output=True
+                )
+                status = json.loads(out.rstrip())
+                list_of_pools = [entry["name"] for entry in status["pools"]]
+                list_of_pools = [
+                    x
+                    for x in list_of_pools
+                    if not (x.startswith(".") or x.startswith("default"))
```

Comment:
we should not delete `device_health_metrics` pool also
```[root@ceph-aarti-kbz-srezbz-node4 ~]# ceph df
--- RAW STORAGE ---
CLASS     SIZE    AVAIL    USED  RAW USED  %RAW USED
hdd    135 GiB  100 GiB  35 GiB    35 GiB      25.96
TOTAL  135 GiB  100 GiB  35 GiB    35 GiB      25.96

--- POOLS ---
POOL                   ID  PGS   STORED  OBJECTS     USED  %USED  MAX AVAIL
device_health_metrics   1    1    876 B        9  2.6 KiB      0     27 GiB
.rgw.root               2   32  1.4 KiB        4   48 KiB      0     27 GiB
default.rgw.log         3   32  3.6 KiB      209  408 KiB      0     27 GiB
default.rgw.control     4   32      0 B        8      0 B      0     27 GiB
default.rgw.meta        5   32    382 B        2   24 KiB      0     27 GiB
pool1                   7   32  1.6 GiB   10.24k   33 GiB  28.88     27 GiB```

Reply:
Added the check